.travis.yml

```diff
@@ -25,7 +25,7 @@ install:
   - export PATH="$HOME/.composer/vendor/bin:$PATH"
 
   # Install Drush.
-  - composer global require drush/drush
+  - composer global require drush/drush:8.*
```

Comment:
This is to fix test autorun on Travis - the default has been drush:9, which has stopped liking our tests.